examples/3dtiles.html

```diff
@@ -80,7 +114,10 @@
 
             // Create a new Layer 3d-tiles For Viewer Request Volume
             // -----------------------------------------------------
-            var $3dTilesLayerRequestVolume = new itowns.GeometryLayer('3d-tiles-request-volume', globe.scene);
+
+            const container = new itowns.THREE.Group();
```

Comment:
You can't use `const` here (examples/* files are restricted to es5 for max compatibility)